language/move-vm/runtime/src/loader.rs

```diff
@@ -659,12 +659,12 @@ impl Loader {
     //   bundle. Basically, this represents the modules appears before `M` in the bundle vector.
     // - the `bundle_unverified` argument tracks the modules that have not been verified when `M`
     //   is being verified, i.e., the modules appears after `M` in the bundle vector.
-    fn verify_module_for_publication(
+    fn verify_module_for_publication<'a>(
```

Comment:
Is the explicit lifetime really necessary? I can't see why the compiler couldn't derive this automatically.